clients/client-python/tests/integration/test_gvfs_with_gcs.py

```diff
@@ -36,12 +36,21 @@
 logger = logging.getLogger(__name__)
 
 
-@unittest.skip("This test require GCS service account key file")
+def oss_is_prepare():
```

Comment:
I think you'd better also unify the python method name, as well as disable messages, some places show "xxx is not prepared", some places show "xxx is not configured".